src/js/util.js

```diff
@@ -844,6 +855,26 @@ var Util;
             } else {
                 el.parentNode.removeChild(el);
             }
+        },
+        wrap: function (el, doc, tag) { // see https://stackoverflow.com/questions/3337587/wrapping-a-set-of-dom-elements-using-javascript
```

Comment:
Could you add some tests on that method ?

Also, add an extra line before the function and move the comment about SO before the function name.